docs/pages/en-us/Getting-Started.md

```diff
@@ -598,12 +388,18 @@ Following is an example for a SampleApp's `viewDocument` event, which takes a si
 
 The name of the event is included in the generated method name. In addition to the custom parameters (document id in this case), the caller must pass the `AuditChannel` object to be used, as well as the tenant id, user id, and correlation id.
 
-The method will throw an exception if the audit event could not be stored for some reason (for example, network failure).
+The method will throw an exception if the audit event could not be stored for some reason (for example, network failure or supplying unsupported tenantId characters).
 
 ### Verification Instructions
 
-Every time an `AuditLog` method is called, a new row is entered into the tenant's audit application table. The following figure shows a tenant's `account_1` schema's `AuditSampleApp` table with an audit event entry with data for an audit event:
+Every time an `AuditLog` method is called, a new application audit event is entered into the tenant's index. Elasticsearch offers a RESTful interface for querying index entries.
+
+The following figure shows all audit event entries belonging to tenant index `00000001_audit`:
+
+![Tenant 00000001_audit all audit event index entries](images/GetTenantIndexAuditEvents.PNG)
+
+The following figure shows all audit event entries belonging to tenant index `00000001_audit` whose `userId` is `JoanneBloggs@yourcompany.com`:
```

Comment:
In the latest commit, reworded to:
"The following Search API operation, `http://<Elasticsearch_Cluser_Node_1>:<Port_Number>/00000001_audit/cafAuditEvent/_search?q=userId:JoanneBloggs@yourcompany.com`, can be run to display the audit event entries belonging to tenantId `00000001` and whose `userId` is `JoanneBloggs@yourcompany.com`:"